graylog2-server/src/main/java/org/graylog2/rest/resources/system/MessagesResource.java

```diff
@@ -77,4 +77,8 @@ public MessagesResource(SystemMessageService systemMessageService) {
                 "messages", messages,
                 "total", systemMessageService.totalCount());
     }
+
+    private int page(int page) {
+        return Math.max(0, page - 1);
+    }
```

Comment:
No, it is related to the general cleanup which involved moving helper methods to the resources where they are used in.